src/test/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtilsUnitTest.java

```diff
@@ -1,33 +1,78 @@
 package org.broadinstitute.hellbender.utils;
 
+import com.google.common.collect.Sets;
 import htsjdk.samtools.SAMSequenceDictionary;
 import htsjdk.samtools.SAMSequenceRecord;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-import org.testng.Assert;
-
-
-import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.*;
-import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.SequenceDictionaryCompatibility.*;
 
+import java.io.File;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
-public class SequenceDictionaryUtilsUnitTest extends BaseTest {
+import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.*;
+import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.SequenceDictionaryCompatibility.*;
+
+public final class SequenceDictionaryUtilsUnitTest extends BaseTest {
 
     private static Logger logger = LogManager.getLogger(SequenceDictionaryUtilsUnitTest.class);
 
+    @Test
+    public void testCreateSequenceDictionaryFromTribbleIndex() throws Exception {
+        final SAMSequenceDictionary dict = IndexUtils.createSequenceDictionaryFromFeatureIndex(new File(getToolTestDataDir(), "test_variants_for_index.vcf"));
+        final Set<String> contigs = dict.getSequences().stream().map(s -> s.getSequenceName()).collect(Collectors.toSet());
+        Assert.assertEquals(contigs, Sets.newHashSet("1", "2", "3", "4"));
+    }
+
+    @Test
+    public void testCreateSequenceDictionaryFromTabixIndex() throws Exception {
+        final SAMSequenceDictionary dict = IndexUtils.createSequenceDictionaryFromFeatureIndex(new File(getToolTestDataDir(), "test_variants_for_index.vcf.bgz"));
+        final Set<String> contigs = dict.getSequences().stream().map(s -> s.getSequenceName()).collect(Collectors.toSet());
+        Assert.assertEquals(contigs, Sets.newHashSet("1", "2", "3", "4"));
+    }
```

Comment:
Move these two test cases to `IndexUtilsUnitTest` since they don't test anything in `SequenceDictionaryUtils`